meta-lxatac-software: labgrid-exporter: more specific usb serial match

hnez opened the pull request:
The less-specific `{{sysfs}}` match also matches on `{{sysfs}}:1.1`, `{{sysfs}}:1.2`, `{{sysfs}}:1.3` etc..
This results in labgrid being rightfully confused about udev events for said devices.

Match on the more specific `{{sysfs}}:1.0` instead.

A side-effect of this change is that the first port is now exported as `lxatac-usb-ports-p1.0` instead of `lxatac-usb-ports-p1`.

This fixes #66.

hnez committed:
meta-lxatac-software: labgrid-exporter: more specific usb serial match

The less-specific '{{sysfs}}' match also matches on '{{sysfs}}:1.1',
'{{sysfs}}:1.2', '{{sysfs}}:1.3' etc..
This results in labgrid being rightfully confused about udev events
for said devices.

Match on the more specific '{{sysfs}}:1.0' instead.

Reported-by: Bastian Krause <bst@pengutronix.de>
Signed-off-by: Leonard Göhrs <l.goehrs@pengutronix.de>

Bastian-Krause commented:
Looks okay from the configuration side, but it will break a lot of existing place matches. Is it worth it?

jluebbe commented:
To keep compatibility, we could just add `:1.0` to the "default" match under `lxatac-usb-ports-p1`.

hnez committed:
meta-lxatac-software: labgrid-exporter: use the same indentation everywhere

The section for additional USB-Serial-Adapter channels was indented slightly
different.

Make indentation consisten.

Signed-off-by: Leonard Göhrs <l.goehrs@pengutronix.de>

hnez commented:
> add `:1.0` to the "default" match under `lxatac-usb-ports-p1`.

That's okay with me. Did that just now.

jluebbe commented:
@Bastian-Krause Are you fine with this?
